aiaccel/hpo/apps/optimize.py

```diff
@@ -114,6 +116,8 @@ def main() -> None:
     """
 
     parser = argparse.ArgumentParser()
+    parser.add_argument("--command")
```

Comment:
```suggestion
    parser.add_argument("command")
```
あばばすみません，ここだけ--command -> commandにお願いできますでしょうか．
この変数は必ず指定するので，位置引数で良いと思います．